### PR TITLE
[DO NOT MERGE] Add configmap to support dual writing to both old gke_container / gce_instance resources and new k8s_container / k8s_node resources.

### PR DESCRIPTION
**Pull request by qingling128**
**What this PR does / why we need it**:
Add configmap to support dual writing to both old gke_container / gce_instance resources and new k8s_container / k8s_node resources.

**Special notes for your reviewer**:
This is not the final PR, just one way of implementing it. We will observe the performance over time and decide how to approach it next.
The easier way to review is to do a diff between `fluentd-gcp-configmap.yaml` and this new configmap file. (Seems GitHub does not supporting branching a file in a PR though...)

**Release note**:
```release-note
Add configmap to support dual writing logs into Stackdriver against both old gke_container / gce_instance resources and new k8s_container / k8s_node resources.
```

### CODE REVIEW

File: cluster/addons/fluentd-gcp/fluentd-gcp-configmap-old.yaml [hunk 1]
```diff
@@ -61,16 +61,18 @@ data:
       # reform.var.log.containers.<POD_NAME>_<NAMESPACE_NAME>_<CONTAINER_NAME>-<CONTAINER_ID>.log
       tag reform.*
       read_from_head true
-      format multi_format
```
Comment (x13n) [inline cluster/addons/fluentd-gcp/fluentd-gcp-configmap-old.yaml]: Why are you  changing a configmap for old resource model in a PR that adds support for both old and new model?

**Comment by qingling128** [comment]
It sounds like we will not be providing a dual write solution from within the agent. If needed, customers will run two instances of Logging Agent to achieve dual write. Closing this PR unless there is any change to this decision.

**Pull request closed without merging**